packages/cli/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@compiled/cli",
+  "version": "0.4.7",
+  "description": "The CSS in JS authoring experience you love without the runtime cost",
+  "author": "Michael Dougall",
+  "license": "Apache-2.0",
+  "homepage": "https://compiledcssinjs.com",
+  "bugs": "https://github.com/atlassian-labs/compiled-css-in-js/issues/new?assignees=&labels=bug&template=bug_report.md",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/atlassian-labs/compiled-css-in-js.git",
+    "directory": "packages/cli"
+  },
+  "main": "./dist/main.js",
+  "module": "./dist/main.js",
+  "types": "./dist/main.d.ts",
+  "scripts": {
+    "start": "ts-node --files src/index.tsx"
+  },
+  "bin": {
+    "compiled-css-in-js": "./dist/index.js"
+  },
+  "files": [
+    "dist",
+    "src"
+  ],
+  "dependencies": {
+    "@compiled/core": "0.4.7",
```

Comment:
make sure this has this dependency declared in the tsconfig project references

Reply:
Its already there 😅